### ISSUE TITLE
Support specifying browsers in the local.browsers api

### ISSUE THREAD

**Issue by panuhorsmalahti** [ekryski/launchpad#58]
Adding an optional function parameter to specify which browsers return would fix the following issue (causing slow test ultimately in web-component-tester):
https://github.com/Polymer/wct-local/issues/21

A comment in the issue explains more clearly the cause of the issue:
https://github.com/Polymer/wct-local/issues/21#issuecomment-174602321

The parameter should be added here:
https://github.com/ekryski/launchpad/blob/master/lib/local/index.js#L59

This is just a quick proposal, I'm open to alternative solutions :).

**Comment by daffl** [comment]
This should already be fixable by specifying the browsers, as added in https://github.com/ekryski/launchpad/pull/52 (also see #51). Additionally, the test could just directly specify what browsers it wants to run (e.g. `launch.firefox` etc.) without calling `.broswers` (which in Windows triggers a lengthy search for all browsers).

**Comment by panuhorsmalahti** [comment]
Yeah, LAUNCHPAD_BROWSERS is (probably) a workaround, but it's an extra unnecessary step for wct users, since the wct configuration file already has a property for specifying the browsers:
https://github.com/Polymer/web-component-tester/blob/master/runner/config.js#L76

**Comment by daffl** [comment]
I would recommend changing the wct tests to use that configuration in `detect` then. `local.browser` only needs to be called if you want a list of all locally installed browsers. If they are already specified it doesn't need to run at all, you just call `launch.firefox`, `launch.chrome` etc. directly. It will error if a user doesn't have the browser installed but I would assume that having the browsers specified in the configuration are a requirement to run the tests.

**Comment by panuhorsmalahti** [comment]
That's a good idea. detect should only use ``local.browsers`` if the user didn't specify which browsers to use.
